docs/android-studio/errors.md

```diff
@@ -7,8 +7,8 @@ title: خطاهای ممکن
 
 این خطاها به دو دسته‌ی زیر تقسیم می‌شوند:
 
-- [خطاهای قبل از اجرای برنامه به هنگام کامپایل](#%D8%AE%D8%B7%D8%A7%D9%87%D8%A7%DB%8C-%D8%AD%DB%8C%D9%86-%D9%86%D8%B5%D8%A8-%D9%88-%D8%B1%D8%A7%D9%87%D8%A7%D9%86%D8%AF%D8%A7%D8%B2%DB%8C)
```

Comment:
These links are referencing your `localhost`. You should only keep anchor hashtags, otherwise anchors will not be resolved.

Reply:
I changed the links as before